lib/parsers.js

```diff
@@ -252,3 +280,23 @@ function tryJsonStringify (obj) {
     return JSON.stringify(obj)
   } catch (e) {}
 }
+
+function getRelativeFileName (filename) {
+  var root = process.cwd()
+  if (root[root.length - 1] !== path.sep) root += path.sep
+  return !~filename.indexOf(root) ? filename : filename.substr(root.length)
+}
+
+/**
+ * Below two functions expects the passed in object to be of type stackframe
+ * https://github.com/stacktracejs/stackframe
```

Comment:
If I understand this correctly, it's just an implementation of the native V8 stack trace API. Better link to that as it's the real source: https://v8.dev/docs/stack-trace-api